<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>560</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>501</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_encabezado">
      <property name="text">
       <string>Encabezado</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_clientes_perdidos">
        <property name="text">
         <string>Clientes Perdidos:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_cp"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_clientes_atendidos">
        <property name="text">
         <string>Clientes Atendidos:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_ca"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_dinero_acumulado">
        <property name="text">
         <string>Dinero Acumulado:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_da"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_reputacion">
      <property name="text">
       <string>Reputacion:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="boton_salir">
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boton_guardar">
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boton_continuar">
        <property name="text">
         <string>Continuar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_continuar</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>continuar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_guardar</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>guardar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_salir</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>salir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>salir()</slot>
  <slot>guardar()</slot>
  <slot>continuar()</slot>
 </slots>
</ui>
